### PR TITLE
Reword python library rst to not use "we"

### PR DESCRIPTION
**Pull request by mayanksharma3**

### CODE REVIEW

File: source/python_library.rst [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-All of our utility functions can be found at `pythonLib/util.py <https://github.com/ManimDSL/ManimDSLCompiler/tree/master/pythonLib/util.py>`_, and all of our shapes are in their own class (i.e. Rectangle_block is in `pythonLib/rectangle.py <https://github.com/ManimDSL/ManimDSLCompiler/tree/master/pythonLib/rectangle.py>`_)
+All of the utility functions can be found at `pythonLib/util.py <https://github.com/ManimDSL/ManimDSLCompiler/tree/master/pythonLib/util.py>`_, and all the shapes are in their own class (i.e. Rectangle_block is in `pythonLib/rectangle.py <https://github.com/ManimDSL/ManimDSLCompiler/tree/master/pythonLib/rectangle.py>`_)
 
 Rectangles
 -----------------
 
-Our Rectangles take in:
-    - A text which is placed inside the rectangle 
+Rectangles's take in:
```
Comment (claralebbos) [inline source/python_library.rst]: shouldn't this just be Rectangles take in?

File: source/python_library.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ To add an additional element, create it, and group it with the VGroup.
 Text Box
 --------------------
 
-Our Text Box takes in:
+Text Box's takes in:
```
Comment (claralebbos) [inline source/python_library.rst]: Text Boxes take in?
Or did you mean the text box's take ins?

Comment (mayanksharma3) [reply source/python_library.rst]: Yeah I meant that but it's better without the comma actually

File: source/python_library.rst [hunk 3]
```diff
@@ -28,7 +28,7 @@ To add an additional element, create it, and group it with the VGroup.
 Text Box
 --------------------
 
-Our Text Box takes in:
+Text Boxes takes in:
```
Comment (claralebbos) [inline source/python_library.rst]: Text Boxes take in?

**Review by claralebbos — APPROVED** [review]

**Pull request merged**